solidity/test/rewards/TestBeaconRewards.js

```diff
@@ -0,0 +1,287 @@
+const { initContracts } = require('../helpers/initContracts')
+const { accounts, contract, web3 } = require("@openzeppelin/test-environment")
+const { createSnapshot, restoreSnapshot } = require("../helpers/snapshot.js")
+const { expectRevert, time } = require("@openzeppelin/test-helpers")
+const crypto = require("crypto")
+const stakeDelegate = require('../helpers/stakeDelegate')
+
+const BN = web3.utils.BN
+const chai = require('chai')
+chai.use(require('bn-chai')(BN))
+const expect = chai.expect
+
+describe('BeaconRewards', () => {
+
+    // accounts
+    const owner = accounts[0]
+    let operators
+    let beneficiaries
+
+    // contracts
+    let rewardsContract
+    let token, stakingContract, operatorContract
+    const groupActiveTime = 5 // from KeepRandomBeaconOperatorBeaconRewardsStub
+    const relayEntryTimeout = 10 // from KeepRandomBeaconOperatorBeaconRewardsStub
+
+    // system parameters
+    const tokenDecimalMultiplier = web3.utils.toBN(10).pow(web3.utils.toBN(18))
+    // 1,000,000,000 - total KEEP supply
+    //   200,000,000 - 20% of the total supply goes to staker rewards
+    //    20,000,000 - 10% of staker rewards goes to the random beacon stakers
+    const totalBeaconRewards = web3.utils.toBN(20000000).mul(tokenDecimalMultiplier)
```

Comment:
If we [allocate](https://github.com/keep-network/keep-core/blob/reward-constants/solidity/test/rewards/TestBeaconBackportRewards.js#L28) 1% of beacon only rewards for May-Sep, then shouldn't the total value from Sep 2020 - Sep 2022 be 20M - 200k = 19 800 000 ?

Reply:
They should be, heh. Please let me recalculate those numbers in a separate PR.